drivers/hmis/app/models/hmis/project_config.rb

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class Hmis::ProjectConfig < Hmis::HmisBase
+  self.table_name = 'hmis_project_configs'
+
+  belongs_to :project, optional: true, class_name: 'Hmis::Hud::Project'
+  belongs_to :organization, optional: true, class_name: 'Hmis::Hud::Organization'
+  validate :project_org_type_xor
+
+  def project_org_type_xor
+    count = [project, organization, project_type].map(&:blank?).count(false)
+    return if count <= 1 # 0 or 1 of these 3 fields can be specified
+
+    errors.add(:base, 'Specify at most one of project, organization, and project type')
+  end
+
+  AUTO_EXIT_CONFIG = 'Hmis::ProjectAutoExitConfig'.freeze
+  AUTO_ENTER_CONFIG = 'Hmis::ProjectAutoEnterConfig'.freeze
+  TYPE_OPTIONS = [AUTO_EXIT_CONFIG, AUTO_ENTER_CONFIG].freeze
+  validates :type, inclusion: { in: TYPE_OPTIONS }
+
+  # TODO: The following 3 methods are repeated code in auto_exit_config.rb.
+  # In the future auto_exit_config.rb will be fully replaced by project_auto_exit_config.rb.
+  def self.default_config
+    find_by(project_type: nil, organization_id: nil, project_id: nil)
+  end
+
+  def self.configs_for_project(project)
+    pc_t = Hmis::ProjectConfig.arel_table
+
+    where(
+      pc_t[:project_id].eq(project.id).
+        or(pc_t[:organization_id].eq(project.organization.id)).
+        or(pc_t[:project_type].eq(project.project_type)),
+    )
+  end
```

Comment:
For consistency we usually prefer to use `scope` over a class method for this kind of thing

```suggestion
  scope :for_project, ->(project) do
    pc_t = Hmis::ProjectConfig.arel_table

    where(
      pc_t[:project_id].eq(project.id).
        or(pc_t[:organization_id].eq(project.organization.id)).
        or(pc_t[:project_type].eq(project.project_type)),
    )
  end
```


I think we need another scope for `active` since the rules can be enabled or not.